var/spack/repos/builtin/packages/brion/package.py

```diff
@@ -39,7 +42,12 @@ class Brion(CMakePackage):
     depends_on('mvdtool ~mpi')
 
     def cmake_args(self):
-        return ['-DDISABLE_SUBPROJECTS=ON']
+        args = ['-DDISABLE_SUBPROJECTS=ON']
+
+        if '+libsonata' in self.spec:
```

Comment:
What is this supposed to do? Since you have no `libsonata` variant, it shouldn't get triggered.